git_machete/docs.py

```diff
@@ -299,17 +300,9 @@
         <b>Usage: git machete g[o] <direction></b>
         where <direction> is one of: `d[own]`, `f[irst]`, `l[ast]`, `n[ext]`, `p[rev]`, `r[oot]`, `u[p]`
 
-        Checks out the branch specified by the given direction relative to the current branch:
```

Comment:
Whoops I wouldn't expect any change to `go`'s help on this PR, probably a rebase glitch